component/admin/libraries/categoryClass.php

```diff
@@ -75,37 +75,39 @@ function store($updateNulls = false){
 			JPluginHelper::importPlugin("jevents");
 			$dispatcher	= JEventDispatcher::getInstance();
 			$set = $dispatcher->trigger('afterSaveCategory', array ($this));
-/*			
+/*
 			$table = JTable::getInstance('Category', 'JTable', array('dbo' => JFactory::getDbo()));
 			if (!$table->rebuild())
 			{
 				throw new Exception( $table->getError(), 500);
 			}
-*/			
+*/
 		}
 		return $success;
 	}
 
 	function getAdminUser(){
-		static $adminuser;
-		if (!isset($adminuser)){
-			$params = JComponentHelper::getParams(JEV_COM_COMPONENT);
-			$adminuser = new  JUser($params->get("jevadmin",62));
-		}
 
 		if (isset($this->_catextra)){
 			if ($this->_catextra->admin>0){
 				$catuser = new JUser();
 				$catuser->load($this->_catextra->admin);
-				return $catuser;
 			}
 		}
 		else if (isset($this->admin) && $this->admin>0){
 			$catuser = new JUser();
-			$catuser->load($this->admin);			
+			$catuser->load($this->admin);
+		}
+
+		// Lets only return once.
+		if (isset($catuser) && $catuser->id > '') {
```

Comment:
!= "" 

I'm sure > will work but it doesn't make sense to say one string it larger than another.